src/monodroid/jni/monodroid-glue.cc

```diff
@@ -634,6 +632,14 @@ JNI_OnLoad (JavaVM *vm, void *reserved)
 	env->DeleteLocalRef (System_javaVersion);
 	env->DeleteLocalRef (System_class);
 
+	if (utils.should_log (LOG_TIMING)) {
+		utils.current_time (total_time.end);
+
+		timing_diff diff;
+		utils.time_difference (total_time, diff);
+		log_info (LOG_TIMING, "JNI_OnLoad: end; elapsed: %lis:%llu::%llu", diff.sec, diff.ms, diff.ns);
```

Comment:
I'm not thrilled with this format.  I think `%lis.%llu` for seconds/milliseconds is "more conventional" (yay), but I'm not sure how `diff.ns` should be represented.  `:` is too easily confused with `hour:minute:second`; `::` is...unique, which is good, but...  ¯\_(ツ)_/¯.

Will `diff.ms` *always* be between 0 and 999?  If so, I think this might be better:

```
%lis.%03llu%llu
```

which would reserve 3 digits for `diff.ms`, and the rest for `diff.ns`.

Reply:
Why is this using `%llu` for `int32_t`?  `%llu` should be for `unsigned long long`, no?

Reply:
The structs use `uint32_t` to keep the values.